Src/BootCamp.Chapter/BootCamp.Chapter.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Was this intentional?

Reply:
No, for some reason I couldn't test my program, so I just right-clicked, went into properties and set it so it would work.

Reply:
Was .net core 2.1 not an option? You might have only installed 3.X, you can download 2.X here: https://dotnet.microsoft.com/download/dotnet-core/2.1

Once you install the SDK, 2.1 should run.

Reply:
Alright, I installed it. I think it's because I am still on VS 2017. I will also install 2019.